jobs/build-arch.Jenkinsfile

```diff
@@ -191,6 +184,15 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}") {
             uploading = false
         }
 
+        def local_builddir = "/srv/devel/streams/${params.STREAM}"
+        def ref = params.STREAM
+        def src_config_commit
+        if (params.SRC_CONFIG_COMMIT) {
+            src_config_commit = params.SRC_CONFIG_COMMIT
+        } else {
+            src_config_commit = shwrapCapture("git ls-remote ${pipecfg.source_config.url} ${ref} | cut -d \$'\t' -f 1")
+        }
+
```

Comment:
I think this is an artifact of a bad rebase

Reply:
Fixed!